src/pages/WorkoutLog.js

```diff
@@ -0,0 +1,76 @@
+import {useState, useEffect} from 'react'
```

Comment:
please remove the use effect, it is blocking the merge from circleCI
```suggestion
import {useState} from 'react'
```